docs/CHANGELOG.asciidoc

```diff
@@ -82,6 +83,48 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+[[release-notes-6.5.4]]
+== {kib} 6.5.4
+coming[6.5.4]
+
+[float]
+[[breaking-changes-6.5.4]]
+=== Breaking changes
+
+See <<breaking-changes, Breaking changes>> for the changes to be aware of 
+when migrating your application from one version of {kib} to another.
+
+[float]
+[[bug-6.5.4]]
+=== Bug fixes
+
+Canvas::
+* Fixes offset mouse capture after scrolling {pull}27096[#27096]
+* Fixes query bug in esdocs {pull}27157[#27157]
+* Sets time to start/end of day when selecting date in the timepicker {pull}27186[#27186]
+* Upgrades TinyMath so that columns with dots work correctly {pull}26659[#26659]
+* Updates the TinyMath reference links {pull}27085[#27085]
+
+Kibana App::
+* Fixes vertical scroll on long legends {pull}23806[#23806]
+* Adds shardTimeout params to body only if not 0 {pull}27217[#27217]
```

Comment:
Is it worth explaining what this change is? I feel like PR name is not very clear. This bug caused Kibana to ignore 0 setting for ES timeout which is supposed to mean "no timeout". Because of this visualizations would return partial data or errors.